circle.yml

```diff
@@ -1,18 +1,71 @@
-dependencies: 
-  cache_directories: 
+# Setup in CircleCI account the following ENV variables:
+# BINTRAY_ACCOUNT
+# BINTRAY_API_KEY
+# DOCKER_USER
+# DOCKER_EMAIL
+# DOCKER_PASSWORD
+general:
+  branches:
+#    only:
+#      - master
+#      - /v[0-9]+(\.[0-9]+)*/
+  artifacts:
+    - /tmp/st2-packages
+
+machine:
+  # Overwrite these ENV variables in parametrized (manual/API) builds
+  environment:
+    ST2_GITURL: https://github.com/StackStorm/st2
+    ST2_GITREV: master
+    ST2_DOCKERFILES_REPO: https://github.com/armab/st2-dockerfiles
+    BUILD_DOCKER: 0
+    DEPLOY_DOCKER: 0
+    DEPLOY_PACKAGES: 0
+  pre:
+    # Paswordless SSH between parallel build nodes
+    - sudo cp /home/ubuntu/.ssh/config /root/.ssh/config
+    # Need latest Docker version for some features to work (CircleCI by default works with outdated version)
+    - |
+      sudo curl -L -o /usr/bin/docker 'http://s3-external-1.amazonaws.com/circle-downloads/docker-1.9.0-circleci'
+      sudo chmod 0755 /usr/bin/docker
+      sudo service docker start
```

Comment:
@dennybaa
Yeah man, it was a new Docker.

This is rspec tests stage: 
[![ubl52ls 1](https://cloud.githubusercontent.com/assets/1533818/11411348/f7f892fc-93d7-11e5-9909-d81e8c2ee90b.png)](https://circleci.com/gh/StackStorm/st2-packages/82)

Doesn't looks like hard to fix, probably adjusting IP/networking related to https://circleci.com/docs/docker#connecting-to-services-outside-of-the-container

Will need time to spend/debug it later when we'll reach stage with docker build/deploy.
Reverting original outdated CircleCI Docker version for now, since deploy stage is inactive for now.